backend/store/key_builder.go

```diff
@@ -68,6 +81,7 @@ func (b KeyBuilder) BuildPrefix(keys ...string) string {
 		if key == "" {
 			continue
 		}
+		//
```

Comment:
```suggestion
```

Reply:
Whoops, left one of my "markers" for things to check out. Good catch!